<launch>
	<node name="Control_Loop" pkg="bulbabot" type="ControlLoop.py" output="screen"/>
	<node name="Main_Program" pkg="bulbabot" type="MainProgram.py" output="screen"/>
</launch>